seahorn/jobs/priority_queue_top/aws_priority_queue_top_harness.c

```diff
@@ -25,27 +25,34 @@ int main(void) {
                                        MAX_ITEM_SIZE));
   assume(aws_priority_queue_is_valid(&queue));
 
-  /* save current state of the container structure */
-  struct aws_array_list old_container = queue.container;
-  struct store_byte_from_buffer old_byte_container;
-  save_byte_from_array((uint8_t *)old_container.data,
-                       old_container.current_size, &old_byte_container);
-
-  /* save current state of the backpointers structure */
-  struct aws_array_list old_backpointers = queue.backpointers;
-  struct store_byte_from_buffer old_byte_backpointers;
-  save_byte_from_array((uint8_t *)old_backpointers.data,
-                       old_backpointers.current_size, &old_byte_backpointers);
-
+  // /* save current state of the container structure */
+  // struct aws_array_list old_container = queue.container;
+  // struct store_byte_from_buffer old_byte_container;
+  // save_byte_from_array((uint8_t *)old_container.data,
+  //                      old_container.current_size, &old_byte_container);
+
+  // /* save current state of the backpointers structure */
+  // struct aws_array_list old_backpointers = queue.backpointers;
+  // struct store_byte_from_buffer old_byte_backpointers;
+  // save_byte_from_array((uint8_t *)old_backpointers.data,
+  //                      old_backpointers.current_size, &old_byte_backpointers);
+
+  sea_tracking_on();
```

Comment:
please add `reset_modified` for memory before calling function under verification. See existing example(s)